src/views/Accessibility/AccessibiltyRequest/Documents/New/index.tsx

```diff
@@ -125,49 +159,179 @@ const New = () => {
           >
             {(formikProps: FormikProps<FileUploadForm>) => {
+              const { errors, setFieldValue, values } = formikProps;
+              const flatErrors = flattenErrors(errors);
               return (
-                <Form onSubmit={formikProps.handleSubmit}>
-                  <h1>
-                    Upload a document to {data?.accessibilityRequest?.name}
-                  </h1>
-                  <Label htmlFor="file-upload">
-                    Choose a document to upload
-                    <FileUpload
-                      id="file-upload"
-                      name="file-upload"
-                      accept=".pdf,.txt"
-                      onChange={onChange}
-                      onBlur={() => {}}
-                      ariaDescribedBy=""
-                    />
-                  </Label>
-
-                  {selectedFile && (
-                    <div className="padding-top-2">
-                      <p>
-                        To keep CMS safe, documents are scanned for viruses
-                        after uploading. If something goes wrong, we&apos;ll let
-                        you know.
-                      </p>
-                      <Button
-                        type="submit"
-                        disabled={
-                          generateURLStatus.loading ||
-                          createDocumentStatus.loading
-                        }
-                      >
-                        Upload document
-                      </Button>
-                    </div>
+                <>
+                  {Object.keys(errors).length > 0 && (
+                    <ErrorAlert
+                      testId="document-upload-errors"
+                      classNames="margin-bottom-4 margin-top-4"
+                      heading="There is a problem"
+                    >
+                      {Object.keys(flatErrors).map(key => {
+                        return (
+                          <ErrorAlertMessage
+                            key={`Error.${key}`}
+                            errorKey={key}
+                            message={flatErrors[key]}
+                          />
+                        );
+                      })}
+                    </ErrorAlert>
                   )}
-                </Form>
+                  <PageHeading>
+                    Upload a document to {data?.accessibilityRequest?.name}
+                  </PageHeading>
+                  <div className="grid-col-9">
+                    <Form onSubmit={formikProps.handleSubmit}>
+                      <Label htmlFor="FileUpload-File">
+                        Choose a document to upload
+                      </Label>
+                      <Field
+                        key="FileUpload-File"
+                        as={FileUpload}
+                        id="FileUpload-File"
+                        name="FileUpload-File"
+                        accept=".pdf,.txt"
```

Comment:
We should confirm the document types. I'd imagine `.doc, .docx` should be allowed too.

Or simply, allow everything (for now)

Reply:
We are allowing everything for now.